azure/azure-preflight.sh

```diff
@@ -264,6 +264,18 @@ function validate_role_assignment() {
 
 validate_role_assignment "$account"
 
+#### Azure User Type Checking
+function validate_user_type() {
+  user_type=$(az ad signed-in-user show -o json | jq .userType | tr -d '"')
+  if [ "$user_type" != "Member" ]; then
+    msg="Current user is a $user_type user, not Member user"
+    store_suggestion "$msg"
+    suggest "$msg" alert
```

Comment:
we need to remove this line temporarily as it breaks the colorful output on some platform.

Reply:
Changed.